src/test/functional/features/check-your-answers.feature

```diff
@@ -0,0 +1,70 @@
+Feature: Check Your Answers
+
+  Background:
+    Given I login
+
+  Scenario: Selecting Wife
+    Given I go to '/your-details'
+    And I clear the form
+    And I click "My wife"
+    And I click "Continue"
+    When I go to '/'
+    Then the page should include "About your marriage"
+    And the page should include "Who are you divorcing?	My wife"
```

Comment:
This might be a pain when it comes to stlying changes in the future as i assume the spaces are taken into consideration.

Reply:
I did look into removing these tab spaces, but they seem to be generated by the [GSD Summary list component](https://design-system.service.gov.uk/components/summary-list/) 😞